spec/models/user_spec.rb

```diff
@@ -122,5 +131,57 @@
         # it { should validate_presence_of(:name) }
       end
     end
+
+    context 'bio validations' do
+      context 'when the number of characters is more than 160' do
+        let(:bio) { Faker::Lorem.paragraph_by_chars(number: 161) }
+        it 'is invalid' do
+          expect(subject).to eq(false)
+        end
+      end
+
+      context 'when the number of characters is between 1 and 160' do
+        let(:random_number) { Faker::Number.between(from: 1, to: 160) }
+        let(:bio) { Faker::Lorem.paragraph_by_chars(number: random_number) }
+        it 'is valid' do
+          expect(subject).to eq(true)
+        end
+      end
+
+      context 'when the bio atribute is empty' do
+        it 'is valid' do
+          expect(subject).to eq(true)
+        end
+      end
+    end
+
+    context 'website validations' do
+      context 'when the website has an invalid format' do
+        let(:website) { 'webexample.com' }
+        it 'is invalid' do
+          expect(subject).to eq(false)
+        end
+      end
+
+      context 'when the website has valid format (http)' do
```

Comment:
You have two context that start with `when the website has valid format`, you could add a `context 'when the website format is valid' do` and inside this context add the different cases of valid format:
```ruby
context 'when the website is valid' do
  context 'when it starts with http' do
    ...
  end
  context 'when it starts with https' do
    ...
  end
end
```